.gitignore

```diff
@@ -1,4 +1,4 @@
 node_modules
 coverage
 .idea
-
+.nyc_output/
```

Comment:
Please extract the changes for setting up code coverage into a standalone pull request.

Also I don't see coveralls results in the checks, do we need to enable loopback-connector-mongodb at coveralls side too?

Reply:
Removed all code coverage stuff.